pkgs/native_assets_builder/test_data/drop_dylib_recording/pubspec.yaml

```diff
@@ -0,0 +1,31 @@
+name: drop_dylib_recording
+description: Generate two dylibs, remove one in linking based on recorded usage.
+version: 1.0.0
+
+publish_to: none
+
+environment:
+  sdk: ^3.0.0
+
+dependencies:
+  logging: ^1.1.1
+  meta: ^1.12.0
+  # native_assets_cli: ^0.7.3
+  native_assets_cli:
+    path: ../../../native_assets_cli/
+  # native_toolchain_c: ^0.5.3
+  native_toolchain_c:
+    path: ../../../native_toolchain_c/
+  record_use: ^0.1.0
+
+dev_dependencies:
+  lints: ^3.0.0
+  test: ^1.24.0
+
+
+
+dependency_overrides:
+  meta:
+    path: ../../../../../../dart-sdk/sdk/pkg/meta/
+  record_use:
+    path: ../../../../../../dart-sdk/sdk/pkg/record_use/
```

Comment:
How does this deal with version skew?

E.g. you land a new version of pkg/record_use that breaks the JSON format.
Then the tests are broken immediately because they rely on a published version.
So that requires you to publish a new `package:record_use` first before it's committed in the SDK, then bump the version here, and then roll dart-lang/native into the SDK.

Thinking about it. Shouldn't this test data live inside the SDK for the time being that `package:record_use` lives in the SDK? Then you can make atomic edits to the test data and format.

Reply:
Probably inside package/record_use/test_data 😄

Reply:
Yeah that's exactly what I am doing right now :D